pkg/util/io/writer.go

```diff
@@ -42,6 +42,11 @@ func (writer *StdWriter) WriteFile(filename string, data []byte, perm os.FileMod
 	return ioutil.WriteFile(filename, data, perm)
 }
 
+const (
+	nsenterPath             = "nsenter"
```

Comment:
Renaming it to `nsenterPath` from `cmd` makes it less clear since it isn't, in fact, a path.